### DISCUSSION TITLE
Unable to see the complete span in Jaeger UI

### DISCUSSION THREAD

**Discussion by shreyash-junio** [jaegertracing/jaeger#3182] [Q&A]
I have installed Jaeger on a K8s cluster using the 
`kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.10/samples/addons/jaeger.yaml  `

Post installation, I only see traces for services with CLuster ip showing up on the Jaeger UI.
I am unable to see any headless service(clusterIP: none) on the Jaeger UI. 

I have enabled istio-injection in all the ns and verified that the sidecar is injected in the pods. However, I still do not see the headless service(type: ClusterIP) on the Jaeger UI. 

What should I do to see these services on Jaeger UI?

**Comment by jpkrohling** [comment]
This would be a better question to the Istio folks, who provided the file you are using.
